modules/RTC/TraceablePeerConnection.js

```diff
@@ -2571,6 +2573,27 @@ TraceablePeerConnection.prototype.setSenderVideoConstraints = function(frameHeig
     if (!localVideoTrack || localVideoTrack.isMuted()) {
         return Promise.resolve();
     }
+
+    const prevPromise = this._lastVideoSenderUpdatePromise;
```

Comment:
Why did you cut this function here instead of also checking if there is a sender and returning early?

Reply:
I did not want to split logical part of taking videoSender and getting params, also in case of later split I would need to either check it once again for videoSender presence (as it might be delayed) or add additional videoSender parameter to _updateVideoSenderParameters. 
Do you think it would be better to move videoSender check in front of _updateVideoSenderParameters and pass it as additional parameter?